test/vcr_python_wrapper.py

```diff
@@ -41,6 +41,21 @@ def domain_query_matcher(r1, r2):
     return r1.query == r2.query
 
 
+def body_json_host_matcher(r1, r2):
```

Comment:
Technically, it's a hostgroup matcher, right?

Reply:
But it's only _used_ for the host tests.

Reply:
How about `body_json_ignore_hostgroup_name_in_search_matcher`

Because it's really neither matching hosts nor hostgroups.

Reply:
`ignore_hostgroup_name_body_matcher` ?
Because it is still a `body_matcher`?

Reply:
Or that, yeah

Reply:
I think,we should call it the `host_body_matcher` to be in line with the `domain_query_matcher`

Reply:
@mdellweg @evgeni This is ready for review.